indico/modules/rb_new/client/js/modules/calendar/actions.js

```diff
@@ -71,7 +71,7 @@ export function fetchCalendar(fetchRooms = true) {
             }
         }
 
-        const params = preProcessParameters(datePicker, ajaxRules);
+        const params = preProcessParameters({...datePicker, myBookings: filters.myBookings}, ajaxRules);
```

Comment:
`myBookings` does not affect the room list so this should not be part of `filters` here (as we use it above to check if the room list must be reloaded as well). You could go for this and then use `roomFilters` instead of filters in the code above:

```js
const {
    calendar: {
        filters: {myBookings, ...roomFilters},
        data: {roomIds},
        datePicker,
    }
} = getState();
```